final_task/rss_reader/text_work_util.py

```diff
@@ -0,0 +1,147 @@
+import feedparser
+import html
+import logging
+import sys
+
+
+def fill_dict_with_info(dict_with_info: dict, all_info: object, limit: str) -> tuple:
+    if limit:
+
+        try:
+            if int(limit) > len(all_info.entries) or int(limit) < 0:
+                limit = int(limit)
+            else:
+                limit = all_info.entries[0:int(limit)]
+        except ValueError:
+            print('\n Should try only integer numbers')
+            sys.exit()
```

Comment:
It is better to have only one exit point from the application. For example you can raise your custom exception here or reraise this exception and handle it in the main function